src/test/java/com/google/gcp_variant_transforms/library/VcfParserImplTest.java

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020 Google LLC
+
+package com.google.gcp_variant_transforms.library;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.google.common.collect.ImmutableList;
+import htsjdk.tribble.TribbleException;
+import htsjdk.variant.vcf.VCFCodec;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.MockitoJUnitRunner;
+
+/**
+ * Units tests for VcfParserImpl.java
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class VcfParserImplTest {
+
+  @InjectMocks
+  VcfParserImpl vcfParserImpl;
+
+  ImmutableList<String> headerLines;
+
+  @Before
+  public void buildHeaderLines() {
```

Comment:
Since I need to use different immutable list, I have removed the setup method and load different list in different method, and moved the string constant to static final string.  
Resolved.